debian/xenial/kubeadm/debian/postinst

```diff
@@ -22,8 +22,8 @@ case "$1" in
         # because kubeadm package adds kubelet drop-ins, we must daemon-reload
         # and restart kubelet now. restarting kubelet is ok because kubelet
         # postinst configure step auto-starts it.
-        systemctl daemon-reload >/dev/null || true
-        systemctl restart kubelet >/dev/null || true
+        systemctl daemon-reload 2>/dev/null || true
```

Comment:
Hi @ixdy, you was right in https://github.com/kubernetes/release/pull/388/commits/5d11d450c091ae87554bcc2bad068fbea49d21fb#r172337046
So fixed. Thanks!